### ISSUE TITLE
Store towns and their registration numbers

### ISSUE THREAD

**Issue by avermeulen** [ggsbv/registration_numbers_webapp#1]
@ggsbv Change your application be able to store towns in the database. Each town should have a name field. When adding a new registration number link the registration number to the appropriate town from a dropdown.

This should be useful: http://mongoosejs.com/docs/populate.html

**Comment by ggsbv** [comment]
Stop giving me more work, Andre !!!!!!!!!!!!!!!!! :D

I'll get right on it, sir. =)

**Comment by avermeulen** [comment]
@ggsbv sorry to say, but there is always more work.

More to learn, more things to explore. Welcome to the journey, one never
really arrives. But it's an adventure of epic proportions.

**Comment by ggsbv** [comment]
@avermeulen 

Agreed. An adventure where the challenge conquered strengthens you, though another challenge awaits, to humble you.

So I'll just go ahead and dive head-first into this pool of humility and get my ass humbled over and over again for the next few days.

wahahaha
